index.js

```diff
@@ -1,45 +1,7 @@
-// nr = not as per eslint:recommended
+export { default as baseConfig } from './configs/base.js';
+export { default as browserConfig } from './configs/browser.js';
+export { default as litConfig } from './configs/lit.js';
+export { default as nodeConfig } from './configs/node.js';
+export { default as reactConfig } from './configs/react.js';
+export { default as testingConfig } from './configs/testing.js';
```

Comment:
The `README` fill will need to be updated with the new instructions for importing and using these.